src/graphq/resolvers.ts

```diff
@@ -0,0 +1,16 @@
+import { runMigration } from '../jobs'
+
+const hello = (): string => 'Hello world!'
+
+const Query = {
+  hello
```

Comment:
did we need this query?

Reply:
tried to remove, we need it as we cannot start the server with empty `Query`